kong/plugins/datadog/schema.lua

```diff
@@ -0,0 +1,8 @@
+return {
+  fields = {
+    host = {type = "string", default = "localhost"},
+    port = {type = "number", default = 8125},
+    metrics = {type = "array", enum = {"request_count", "latency", "request_size", "status_count"}, default = {"request_count", "latency", "request_size", "status_count"}}, 
+    timeout = {type = "number", default = 10000}
```

Comment:
Should be `required` because you always call `settimeout` without checking if `conf.timeout` exists. I think it makes sense to have it `required` too anyways.